assets/scss/colours.scss

```diff
@@ -1,28 +0,0 @@
-$color_snow-white: #FFFFFF;
-$color_black: #000000;
-$color_charcoal-black: #0B0C0C;
-$color_metal-grey: #535A5E;
-$color_concrete-grey: #B1B4B6;
-$color_light-grey: #E8EDEE;
-$color_grey-1: #425563;
-$color_grey-2: #768692;
-$color_grey-3: #aeb7bd;
-$color_grey-4: #d8dde0;
-$color_grey-5: #f0f4f5;
-$color_blue: #005eb8;
-$color_southsea-blue: #1976D2;
-$color_bright-blue: #0072CE;
-$color_marine-blue: #0D47A1;
-$color_bluebell-blue: #403DD0;
-$color_navy-blue: #003087;
-$color_dark-blue: #0C223F;
-$color_lavendar-purple: #7E57C2;
-$color_yellow: #ffeb3b;
-$color_sunflower-yellow: #FFE600;
-$color_warm-yellow: #FFB81C;
-$color_red: #da291c;
-$color_teal: #336C83;
-$color_light-teal: #00a19a;
-$color_aqua-green: #00a499;
-$color_dark-green: #006747;
-$color_green: #007F3B;
```

Comment:
Just wanted check we dont need some of these colours as bluebell was being used for the default link colour  and lavendar for visited link

Reply:
Where these were needed for link colours etc, they've been re-assigned elsewhere.  But I have retained these colours in a comment in `page-colours.scss` just in case we want to know what they were.